dipy/denoise/noise_estimate.py

```diff
@@ -138,60 +250,8 @@ def piesno(data, N=1, alpha=0.01, l=100, itermax=100, eps=1e-5, return_mask=Fals
         # Remember the biggest omega array as giving the optimal
         # sigma amongst all initial estimates from phi
         if omega_size > max_length_omega:
-            pos, max_length_omega, best_mask = num, omega_size, idx
+            pos, max_length_omega = num, omega_size
 
         sigma[num] = sig
         mask[num] = idx
-
-    if return_mask:
```

Comment:
does this PR actually remove the old noise estimation stuff? It's still sueful for T1 imaging if that's the case.